synonyms/synonyms.js

```diff
@@ -0,0 +1,38 @@
+import { createDirectus, readItems, rest, staticToken } from "@directus/sdk";
+import words from "an-array-of-english-words" assert { type: "json" };
+import "dotenv/config.js";
+import * as fs from "fs";
+const getDirectusClient = createDirectus(process.env.DIRECTUS_URL)
+  .with(staticToken(process.env.DIRECTUS_STATIC_TOKEN))
+  .with(rest());
+
+const audioData = await getDirectusClient.request(
+  readItems("audios", {
+    fields: ["id", "title", "topics"],
+    filter: { status: { _eq: "active" } },
+  })
+);
+
+let cmsData = "";
+Object.values(audioData).forEach((value) => {
+  const title = value["title"];
+  const topics = value["topics"];
+  const allWords = title.concat(topics);
+  const wordlist = allWords.match(/\b\w+\b/g).join("\n");
```

Comment:
Here you join all words into a string and later you split that string back. Why not to accumulate the words here into an array directly? Or even better a Set. You can use this new method that id available in Node 22: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/union. Just make sure your local Node version is not lower.